### ISSUE TITLE
Discovery: Formatting of accessible phone numbers in rich text editors

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#13745]
## Description
Editors are (and should be) able to add phone numbers into rich text fields. However, there is currently no simple, easy way for the editors to format these that follows the [VA style guidelines](https://design.va.gov/content-style-guide/dates-and-numbers#phone-numbers) and that makes them accessible. Currently the only way to do this is to add HTML into the source view of the editor - and that is not a level of skill that we should expect our editors to have.

## Proposed solution
The [Editor Advanced link](https://www.drupal.org/project/editor_advanced_link) module is available that could assist editors to correctly and accessible format phone numbers.

- This was tested with simplytest.me
  - It does still need to be tested with the VA implementation of Drupal 10 to confirm compatibility
- This will add an aria-label field to the link widget in the rich text field
  - There would be no requirement or validation on this field so editors would be able to add phone numbers without the aria-label or with an incorrectly formatted aria-label.
  - There would not be any restriction on the aria-label field so editors could potentially use it for other types of links, which would not be correct.
- There are other formatting requirements/guidelines besides the aria-label (adding +1 in the href for example) that are needed that would not be addressed with this module.
- There is no guarantee that editors will use the new field.
- Communication and training will be needed for editors to information of proper usage of the module.

Note: To alleviate some of the risks of improper usage, a potential work around could be to create a new editor configuration and then only add this to one content type/product and see what happens.

### Additional Info
From the previous module research:

- I didn't find any modules that had a link type option, something that would let editors choose which link type they were creating (i.e. tel, mailto, etc.)
- I didn't find a more specific phone number widget for the CKEditor, anything that was for tel links only not all link types.
- There is the possibly that we could  make our own custom module if we felt this would be helpful for us and our editors. Individual phone number and ext field and then could we auto process this into the correct code in the WYSIWYG with the proper aria-label and href? With out research, this might be much more work than we need and not worth the effort.

## Acceptance Criteria
- [ ] Confirm with PO if Editor Advanced link module is appropriate solution (vs. custom module)
- [ ] Determine implementation plan (i.e. rollout to all or specific rich text editors)
- [ ] Create change management plan including list of training materials needed

**Comment by laflannery** [comment]
After meeting with @EWashb and @productmike , the possible cons and editor confusion outweigh the small benefit this module would give us. We not going to go forward (tough noogies) but I made 3 new tickets:

- Audit of phone numbers, how many are wrong (#13949)
- Screen reader audit, which might read these right without a label (#13951)
- Discovery ticket for custom module (#13952)
